src/app/containers/suggestion-of-activities/suggestion-of-activities.component.html

```diff
@@ -4,28 +4,24 @@
   </div>
   <alg-error
     *ngIf="state.isError"
+    class="light"
+    icon="ph-duotone ph-warning-circle"
+    buttonStyleClass="alg-button primary refresh-button light-border"
     i18n-message message="Error while loading the activities"
     [showRefreshButton]="true"
     refreshButtonType="refresh"
-    (refresh)="refresh()"
+    (refresh)="refresh($event)"
   ></alg-error>
-  <div *ngIf="state.isReady && state.data.length === 0" i18n>
+  <div class="empty-list" *ngIf="state.isReady && state.data.length === 0" i18n>
     There are no activities
   </div>
-  <p-table
-      class="alg-table --overlay-panel"
-      [value]="state.data"
-      *ngIf="state.isReady && state.data.length > 0"
-  >
-    <ng-template pTemplate="body" let-item>
-      <tr>
-        <td><span class="type">{{ item.activity.type }}</span></td>
-        <td>
-          <a class="alg-link dots" [routerLink]="item.activity | rawItemRoute | url">
-            {{ item.activity.string.title }} ({{ item.name }})
-          </a>
-        </td>
-      </tr>
-    </ng-template>
-  </p-table>
+  <ul class="list" *ngIf="state.isReady && state.data.length > 0">
+    <li class="list-item" *ngFor="let item of state.data">
+      <span class="item-type">{{ item.activity.type }}</span>
```

Comment:
"Chapter/Task > ... " is a bit strange.

 But anyway, thinking about that, I don't think mentioning the type is really useful in this case. So just remove the type and chevron.

Reply:
Fixed